configure

```diff
@@ -8,8 +8,22 @@ pushd `dirname $0` #> /dev/null
 SOURCE_BASE_DIR=`pwd -P`
```

Comment:
Could this include spaces? Do we need quotes around each use of `$SOURCE_BASE_DIR` as well?